src/compiler/diagnosticMessages.json

```diff
@@ -2477,5 +2477,10 @@
     "A type assertion expression is not allowed in the left-hand side of an exponentiation expression. Consider enclosing the expression in parentheses.": {
       "category": "Error",
       "code": 17007
+    },
+    "Potentially wrong empty statement in THEN clause.": {

```

Comment:
I try to follow "[preconditions] [cannot / must be] [postconditions]" or "[noun] [cannot / must be] [adjective phrase]"

Here, `The body of an 'if' statement cannot be the empty statement` ?